src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -294,13 +296,87 @@ public object Run(int splitId, object input)
         }
     }
 
+    internal class ArrowBasedCommandExecutor : SqlCommandExecutor
```

Comment:
`abstract`?

Then you don't need to override `ExecuteCore`.

Reply:
Yup.